sound/soc/intel/boards/sof_sdw.c

```diff
@@ -752,6 +752,14 @@ static const struct dmi_system_id sof_sdw_quirk_table[] = {
 		},
 		.driver_data = (void *)(SOC_SDW_PCH_DMIC),
 	},
+	{
+		.callback = sof_sdw_quirk_cb,
+		.matches = {
+			DMI_MATCH(DMI_SYS_VENDOR, "Google"),
```

Comment:
Are we still remaining the SYS_VENDOR string in coreboot? Perhaps DMI_PRODUCT_FAMILY or DMI_MANUFACTURER instead?

Reply:
I hope manufacturer string in coreboot is used for SYS_VENDOR. We can use DMI_PRODUCT_FAMILY too, however, we are sticking to the way we defined in MTL platform for REX boards.